packages/api/src/express/chatRouter.ts

```diff
@@ -1,77 +1,102 @@
 import express from 'express';
 import { Experimental_StdioMCPTransport as StdioMCPTransport } from 'ai/mcp-stdio';
 import {
-    experimental_createMCPClient as createMCPClient,
-    InvalidToolArgumentsError,
-    NoSuchToolError,
-    ToolExecutionError,
-    streamText,
+  experimental_createMCPClient as createMCPClient,
+  InvalidToolArgumentsError,
+  NoSuchToolError,
+  ToolExecutionError,
+  streamText,
 } from 'ai';
 import { anthropic } from '@ai-sdk/anthropic';
 import { defaultLogger } from '@unchainedshop/logger';
+import rateLimit from 'express-rate-limit';
```

Comment:
Remove this from the api package, it's the user that needs to add rate limiting and such kind of stuff as we don't know it what environment the system runs and/or if it's behind some reverse proxy etc. Anyways, rate limiting should be done on a reverse proxy level if possible.